test/sequential/bootstrap.checkActivatePool.test.ts

```diff
@@ -0,0 +1,209 @@
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
+
+let testUser1: User;
+let sudo: User;
+let keyring: Keyring;
+let bootstrapPhase: any;
+let bootstrapCurrency: any;
+let bootstrapPool: any;
+let eventResponse: EventResult;
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+const waitingPeriod = 15;
+const bootstrapPeriod = 30;
+const whitelistPeriod = 10;
+const bootstrapAmount = new BN(10000000000);
+
+async function checkingPool() {
+  const creatingPool = await (
+    await getMangataInstance()
+  ).createPool(
+    testUser1.keyRingPair,
+    MGA_ASSET_ID.toString(),
+    bootstrapAmount,
+    bootstrapCurrency.toString(),
+    bootstrapAmount
+  );
+  eventResponse = getEventResultFromMangataTx(creatingPool);
+  expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicFailed);
+}
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+
+  keyring = new Keyring({ type: "sr25519" });
+
+  const api = getApi();
+  // setup users
+
+  sudo = new User(keyring, sudoUserName);
+  testUser1 = new User(keyring);
+
+  [testUser1] = setupUsers();
+
+  const bootstrapCurrencyIssue = await sudoIssueAsset(
+    sudo.keyRingPair,
+    toBN("1", 20),
+    sudo.keyRingPair.address
+  );
+  const bootstrapEventResult = await getEventResultFromMangataTx(
+    bootstrapCurrencyIssue,
+    ["tokens", "Issued", sudo.keyRingPair.address]
+  );
+  const bootstrapAssetId = bootstrapEventResult.data[0].split(",").join("");
+  bootstrapCurrency = new BN(bootstrapAssetId);
+
+  // check that system is ready to bootstrap
+  bootstrapPhase = await api.query.bootstrap.phase();
+  if (bootstrapPhase.toString() === "Finished") {
+    const bootstrapFinalize = await finalizeBootstrap(sudo);
+    eventResponse = getEventResultFromMangataTx(bootstrapFinalize);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  }
+
+  bootstrapPhase = await api.query.bootstrap.phase();
+  expect(bootstrapPhase.toString()).toEqual("BeforeStart");
+});
+
+test("bootstrap - Check that we can not create a pool for the bootstrap token after bootstrap was declared", async () => {
+  const api = getApi();
+  await setupApi();
+
+  bootstrapPhase = await api.query.bootstrap.phase();
+  if (bootstrapPhase.toString() === "Finished") {
+    const bootstrapFinalize = await finalizeBootstrap(sudo);
+    eventResponse = getEventResultFromMangataTx(bootstrapFinalize);
+    expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+  }
+
+  bootstrapPhase = await api.query.bootstrap.phase();
+  expect(bootstrapPhase.toString()).toEqual("BeforeStart");
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintToken(bootstrapCurrency, testUser1), // transferAll test
+    Assets.mintToken(bootstrapCurrency, sudo), // transferAll test
+    Assets.mintNative(testUser1)
+  );
+
+  await sudo.mint(bootstrapCurrency, testUser1, toBN("1", 20));
```

Comment:
You can avoid this minting with the ones in L82. 
Put the user address instead of sudo and remove this one :)